### DISCUSSION TITLE
GETNEWLIBL

### DISCUSSION THREAD

**Discussion by JMARFINK** [codefori/vscode-ibmi#1910] [Q&A]
In the process of setting up the CODE for IBM i, I receive this error:  "Unable to install GETNEWLIBL. See Code for IBM i output."
Where is this 'output' and what is this error.

**Comment by worksofliam** [comment]
Please share the contents of your Code for IBM i output channel after the error appears.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Comment by JMARFINK** [comment]
Hi,
That's what I am trying to find out.  Where is my 'contents of Code for IBM I channel'?

Jack Marfink
Senior Application Developer
(919) 605-2175 Work
or
(540) 815-7398 Home
[Southern Glazers_Signature_Enterprise Technology Partners]

**Comment by JMARFINK** [comment]
Hi again.
OK, I think I found it:  Apparently I am missing some files.
/home/JMARFINK: /QOpenSys/usr/bin/qsh
system  "QSYS/CPYFRMSTMF FROMSTMF('/tmp/vscodetemp-O_ys9D3qQe') TOMBR('/QSYS.LIB/ILEDITOR.LIB/QTOOLS.FILE/GENCMDXML.MBR') MBROPT(*REPLACE) STMFCCSID(1208) DBFCCSID(*FILE)"
{
    "code": 1,
    "signal": null,
    "stdout": "",
    "stderr": "CPFA0A9:  Object not found.  Object is /QSYS.LIB/ILEDITOR.LIB/QTOOLS.FILE/GENCMDXML.MBR.\nCPFA095:  Stream file not copied.\nCPC2206:  Ownership of object QZSHSYSTEM in QTEMP type *USRSPC changed."



Jack Marfink
Senior Application Developer
(919) 605-2175 Work
or
(540) 815-7398 Home
[Southern Glazers_Signature_Enterprise Technology Partners]

**Comment by sebjulliand** [comment]
Hi @JMARFINK ,
could you check what's the temporary library in your connection's settings?
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/fbaa4664-da93-4af9-a5b2-aedb6e24fc48)

This error sometimes happens if you leave the default libarry (i.e. `ILEDITOR`) on which you may not have enough authority.
If it's the case, change it for another library (in which you can create objects) and reconnect.
